### DISCUSSION TITLE
Interference between active and inactive zwave networks?

### DISCUSSION THREAD

**Discussion by murkumurdok** [zwave-js/zwave-js#7298] [Q&A] [answered]
This is more of a technical question about the way zwave neworks work generally as opposed to zwave-js itself. I have a Qolsys IQ Panel 4 as my alarm system hub, which has its own Zwave radio for home automation capability.  For the past several years I have just kept the panel's Zwave radio disabled because I didn't want it to interfere with my existing Zwave-JS network on Home Assistant and had no intention on creating a seperate Zwave hub on the panel.  However, a recent firmware update on the panel now flags the disabled Zwave radio as a "problem" and sends me very annoying notifications on the panel, with no ability to turn them off.  My question is this:  Will turning on the IQ Panel's Zwave radio interfere with my existing Zwave network in any way if I just don't add any devices to it?  If it could cause interference, what should I look for in the Zwave-JS logs to confirm whether some interference is happening.  For what it's worth, the panel and my Zooz Zwave stick are about 25 feet apart on different floors of the house, and I never noticed any issues before because I disabled the Zwave radio on the panel as soon as I installed it.  Thanks for your help.

**Comment by AlCalzone** [comment, marked as answer]
If there are no devices, there will be no traffic that can interfere with another network.

**Reply by murkumurdok** [reply]
That's what I thought but wanted to confirm, thanks very much
